### PR TITLE
Remove interface

### LINKED ISSUE #175: Separate interface into a separate Github repository
Given the relative stability of the API, it may simplify development and deployment to have the interface code separated into a distinct repository, something like "microsetta-interface".

### PR DESCRIPTION
**Pull request by wasade**
Fixes #175 
cc biocore/microsetta-interface#1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,10 +24,6 @@ Install the JSON Web Tokens library with cryptography support:
 
 `pip install pyjwt[crypto]`
 
-Install redis:
```
Comment (dhakim87) [inline README.md:27]: Did this make it into the migrated README?

File: microsetta_private_api/api/_consent.py [hunk 2]
```diff
@@ -13,10 +13,9 @@ def render_consent_doc(account_id, language_tag, consent_post_url, token_info):
...
     localization_info = localization.LANG_SUPPORT[language_tag]
-    consent_html = render_template(
-        "new_participant.jinja2",
-        tl=localization_info[localization.NEW_PARTICIPANT_KEY],
-        lang_tag=language_tag,
-        post_url=consent_post_url
-    )
-    return jsonify({"consent_html": consent_html}), 200
+    payload = {'lang_tag': language_tag,
+               'post_url': consent_post_url}
```
Comment (dhakim87) [inline microsetta_private_api/api/_consent.py:17]: What?  Why are we just echoing language_tag and consent_post_url?  Do we need to update the yaml to remove consent_post_url?  Will language_tag be going into the localization_info lookup in the future?

File: README.md (hunk 1 above)
Comment (wasade) [reply README.md:27]: No need, just added it to conda reqs so the user doesn't need to explicitly install it

File: microsetta_private_api/api/_consent.py (hunk 2 above)
Comment (wasade) [reply microsetta_private_api/api/_consent.py:17]: Good catch. I figured `lang_tag` was used in the template for rendering but it does not seem so. The `post_url` is of course used

File: microsetta_private_api/api/microsetta_private_api.yaml [hunk 3]
```diff
@@ -194,9 +194,62 @@ paths:
...
+                    type: string
+                  consent_your_child:
+                    type: string
+                  assent_13_17:
+                    type: string
+                  text_i_have_read_1:
+                    type: string
+                  consent_18:
+                    type: string
+
```
Comment (dhakim87) [inline microsetta_private_api/api/microsetta_private_api.yaml:252]: Do the consent docs work post migration?  It looks like this collapses down the dictionary of values of the jinja2 variable "tl", but I didn't see all references updated on the other side.

Comment (wasade) [reply microsetta_private_api/api/microsetta_private_api.yaml:252]: yes. Which reference is missing?

Comment (wasade) [reply microsetta_private_api/api/microsetta_private_api.yaml:252]: it does seem that openapi (or connexion) doesn't care about case of the keys though

File: microsetta_private_api/api/_consent.py (hunk 2 above)
Comment (dhakim87) [reply microsetta_private_api/api/_consent.py]: post_url is just echoed from whatever the interface passed up to the api here, that seems strange to me.

File: microsetta_private_api/api/microsetta_private_api.yaml (hunk 3 above)
Comment (dhakim87) [reply microsetta_private_api/api/microsetta_private_api.yaml:252]: I have no idea how jinja2 managed to pull the correct value for {{ post_url }} given my current read, but I'm glad that it does :D

File: README.md (hunk 1 above)
Comment (dhakim87) [reply README.md:27]: Did we tell the user to start the redis server in the old readme?  I'm not sure.

Comment (wasade) [reply README.md:27]: nope

File: microsetta_private_api/api/microsetta_private_api.yaml (hunk 3 above)
Comment (wasade) [reply microsetta_private_api/api/microsetta_private_api.yaml:248]: it didn't...

**Pull request merged**